Sources/App/Controllers/PostController.swift

```diff
@@ -44,14 +44,13 @@ class PostController: RouteCollection {
     }
     
     func postsHandler(_ req: Request) throws -> Future<[PostItem]> {
-        _ = try req.requireAuthenticated(User.self)
-        
-        return PostItem.query(on: req).all()
+        try req.authorizedUser().flatMap { _ in
```

Comment:
I think this is not needed here 🤔 Going to remove it, as the user is already authorized in the middleware, no need to get them from DB here

Reply:
Ya looks like it :+1:  Good catch